scripts/reclone-hardlinked-deps.py

```
@@ -0,0 +1,281 @@
+                    help="additional worktree root to scan (repeatable)")
+    ap.add_argument("--dry-run", action="store_true", help="report only; change nothing")
+    ap.add_argument("--quiet", action="store_true", help="summary only")
+    args = ap.parse_args()
+
+    clonefile = load_clonefile()
+    if clonefile is None:
+        print("Not macOS/APFS — hardlink imports are correct here; nothing to do.")
+        return 0
+
+    if not args.dry_run:
+        if (proc := installer_running()) is not None:
```

Ruff (0.16.2)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)